grails-app/conf/application.yml

```diff
@@ -183,43 +190,65 @@ dataSource:
     pooled: true
     jmxExport: true
     driverClassName: com.mysql.cj.jdbc.Driver
-    username: 'openboxes'
-    password: 'openboxes'
+    dbCreate: none
     dialect: org.hibernate.dialect.MySQL57InnoDBDialect
-    logSql: false
-    formatSql: false
+    factory: org.apache.tomcat.jdbc.pool.DataSourceFactory
+    logger: com.mysql.cj.jdbc.log.StandardLogger
+    type: org.apache.tomcat.jdbc.pool.DataSource
     properties:
+        # from https://docs.grails.org/3.3.16/guide/conf.html#dataSource
         jmxEnabled: true
-        initialSize: 10
-        maxActive: 100
+        initialSize: 5
+        maxActive: 50
         minIdle: 5
         maxIdle: 25
         maxWait: 10000
-        maxAge: 14400
+        maxAge: 10 * 60000
```

Comment:
Can you provide justification for these changes?

Reply:
`maxAge` is a few hundred milliseconds less than `validationInterval`, which, I believe, means that the db pool will never send a heartbeat ping to keep a connection alive. Instead, 4 times per minute, all unused db connections are destroyed and replaced with new ones. This seems like unnecessary churn, so I changed this to the value recommended by the Grails documentation. But, it's not strictly needed to get a clean deploy -- I think I'll pull all these config changes to a separate PR where we can discuss at leisure.